content/content.go

```diff
@@ -43,7 +44,7 @@ type ruleIDAndErrorKey struct {
 // It's thread safe
 type RulesWithContentStorage struct {
 	sync.RWMutex
-	rulesWithContent map[ruleIDAndErrorKey]*types.RuleWithContent
+	rulesWithContent map[ruleIDAndErrorKey]*local_types.RuleWithContent
```

Comment:
why did you change this?

Reply:
Because I use local `RuleWithContent` which is used only in smart-proxy